components/localhost_permission_allowlist/browser/BUILD.gn

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//brave/build/config.gni")
+
+static_library("browser") {
+  sources = [
+    "localhost_permission_allowlist_service.cc",
+    "localhost_permission_allowlist_service.h",
```

Comment:
maybe drop `_service` from the name

Reply:
and maybe also name component "localhost_permission"? or something. For future extensibility

Reply:
It made sense to me to keep the `_service` since it's a service similar to the other ones in `BraveBrowserProcessImpl::StartBraveServices()` and we start it over there.

I can remove the `_allowlist` part.

Reply:
it's more similar to `brave_ads::ResourceComponent` then :) I'm not really insisting, but "service" is typically either keyed service, or something that has more sophisticated logic

Reply:
Yeah fair, I renamed it.